ols/src/query_helpers/chat_history.py

```diff
@@ -0,0 +1,46 @@
+"""Produce chat history object using list of Conversations."""
+
+from typing import Optional
+
+from langchain.schema import ChatMessage as langchainCM
+from llama_index.llms import ChatMessage, MessageRole
+
+from ols.src.cache.conversation import Conversation
+
+
+def get_llama_index_chat_history(
+    conversations: Optional[list[Conversation]],
+) -> list[ChatMessage]:
+    """Produce list of ChatMessage object using list of Conversations for llama_index."""
+    chat_history: list[Conversation] = []
+    if conversations is None or len(conversations) < 1:
+        return chat_history
```

Comment:
this isn't the right type.  The function returns a list[ChatMessage]. but chat_history is a list[Conversation]